dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -10,9 +10,14 @@ var addClickTouchEvent = function(element, handler) {
 };
 
 function onURLBlocked(base_url, callback) {
-  $("<img width=0 height=0>")
+  $('<img width="0" height="0">')
```

Comment:
In general it's good to avoid them, so that's not a bad idea. Worth spending some extra time testing if you do that.